src/receivers/HTTPModuleFunctions.ts

```diff
@@ -268,3 +268,7 @@ export interface ReceiverUnhandledRequestHandlerArgs {
   request: IncomingMessage;
   response: ServerResponse;
 }
+
+export interface ReceiverInvalidRequestSignatureHandlerArgs {
+  error: Error;
+}
```

Comment:
Unsure if this is the **most** ideal place for an export - it seems fine but open to other suggestions!

Placing it here does also raise question around if we want to include these checks for `this.invalidRequestSignatureHandler` in other receivers like the `ExpressReceiver` and `HTTPReceiver`. I'm thinking now'd be a good time to do so to avoid confusion around similar `App` initializations with a different `receiver`.